processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/test.ldst1.asm (1 of 2)
processing file asmFiles/test.ldst2.asm (2 of 2)
--- memsim.hex	2021-09-30 20:46:26.273532801 -0400
+++ memcpu.hex	2021-09-30 20:46:26.218530725 -0400
@@ -13,7 +13,7 @@
 :040200000000733750
 :0402010000002701D1
 :0402020000001337AE
-:04020300FEEDBEEF5F
+:040203000000BEEF4A
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
